lib/commonmarker/renderer.rb

```diff
@@ -97,5 +97,32 @@ def escape_href(str)
     def escape_html(str)
       @node.html_escape_html(str)
     end
+
+    def tagfilter(str)
+      if @tagfilter
+        str.gsub(
+          %r{
+            <
+            (
+            title|textarea|style|xmp|iframe|
+            noembed|noframes|script|plaintext
+            )
+            (?=\s|>|/>)
+          }x,
+          '&lt;\1')
+      else
+        str
+      end
+    end
+
+    def sourcepos(node)
+      if @opts & CommonMarker::Config::Render::SOURCEPOS != 0
```

Comment:
Rather than the `if` indentation I'd rather this bail early:

``` ruby
return "" unless @opts & CommonMarker::Config::Render::SOURCEPOS.zero?
...
```